.github/workflows/release.yml

```diff
@@ -8,6 +8,18 @@ jobs:
     name: Create Release
     runs-on: ubuntu-latest
     steps:
+      # v12
+      - uses: cachix/install-nix-action@v12
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
```

Comment:
why do we need this?

Reply:
because we're installing with nix-build the next line